util/chunk/column.go

```diff
@@ -85,6 +85,29 @@ func (c *column) appendNullBitmap(on bool) {
 	}
 }
 
+func (c *column) appendMultiSameNullBitmap(on bool, num int) {
```

Comment:
s/ on/ notNull ?

Reply:
I see `appendNullBitmap` also use `on`, so, both use `on` or `notNull` ?

Reply:
I prefer to both use `on`.